cypress/utils.js

```diff
@@ -1,5 +1,7 @@
 export const loadPage = (url) => {
 	it(`should load the page: ${url}`, () => {
+		cy.clearLocalStorage();
```

Comment:
Consent state is stored both in local storage and cookies in the unified version https://documentation.sourcepoint.com/web-implementation/general/cookies-and-local-storage